.github/scripts/update-quality-gate-db.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Run your command and capture the output
+output=$(go run ./cmd/grype/main.go db list)
+
+# Extract the first instance of URL using grep, cut, and sed to trim leading whitespace
+url=$(echo "$output" | grep -m 1 -o 'URL: .*' | cut -d' ' -f2- | sed 's/^[[:space:]]*//')
```

Comment:
if we are ok with `jq` then we could use `grype db list -o json` or `-o raw` and make this a little simpler

Reply:
FWIW I thought I posted this but it was just pending 🤦 I think this could work:
```
url=$(go run ./cmd/grype db list -o json | jq '.[0].url')
```
... and `jq` is available on the runners by default